.github/ISSUE_TEMPLATE/Enhancement.md

```diff
@@ -2,7 +2,9 @@
 name: Enhancement
 about: Suggest an enhancement for existing odo functionality
 ---
-[kind/Enhancement]
+
+/kind code refactoring
```

Comment:
I don't think that "code refactoring" and Enhancement are the same

Reply:
Yes, I know, but there is no such thing as the "enhancement" label :(. What alternative do you suggest?

Reply:
We can either create it. Or I would even consider getting rid fo enhancement altogether.
Is't enchantment kind of small new feature?

Reply:
Yeah.. that's what I thought too haha. I'm going to remove it.